cmd/nomos/status/client.go

```diff
@@ -168,6 +168,9 @@ func (c *ClusterClient) clusterStatus(ctx context.Context, cluster, namespace st
 
 	if namespace != "" {
 		c.namespaceRepoClusterStatus(ctx, cs, namespace)
```

Comment:
`c.namespaceRepoClusterStatus` lists all RepoSyncs in the namespace no matter if the namespace is c-m-s or not.
Do you need it when namespace is c-m-s?

Reply:
the logic has been update to only show root for c-m-s ns